src/Protocol/Diaspora.php

```diff
@@ -2376,15 +2382,16 @@ private static function addReshareActivity($item, $parent_message_id, $guid, $au
 	/**
 	 * Processes a reshare message
 	 *
-	 * @param array  $importer Array of the importer user
-	 * @param object $data     The message object
-	 * @param string $xml      The original XML of the message
+	 * @param array  $importer  Array of the importer user
+	 * @param object $data      The message object
+	 * @param string $xml       The original XML of the message
+	 * @param int    $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.